.env-cmdrc.json

```diff
@@ -13,6 +14,7 @@
     "REACT_APP_WSS_URL": "wss://h.bitfinex.com/ws/",
     "REACT_APP_UFX_PUBLIC_API_URL": "https://api-pub.bitfinex.com",
     "REACT_APP_UFX_API_URL": "https://api.bitfinex.com",
+    "REACT_APP_UFX_PLATFORM_URL": "https://www.bitfinex.com",
```

Comment:
@tarcisiozf lets rename it to not use UFX (may be REACT_APP_HONEY_PLATFORM_URL), because UFX_* envs are used for internal package ufx-ui, while this will be only used for bfx-hf-ui-core repo.